src/jit/emitarm64.cpp

```diff
@@ -6220,6 +6335,102 @@ void emitter::emitIns_S_R(instruction ins, emitAttr attr, regNumber reg1, int va
 
 /*****************************************************************************
  *
+ *  Add an instruction referencing consecutive stack-based local variable slots and two registers
+ */
+void emitter::emitIns_S_S_R_R(
+    instruction ins, emitAttr attr1, emitAttr attr2, regNumber reg1, regNumber reg2, int varx, int offs)
+{
```

Comment:
Well that change should be cross-platform as emitIns_S_R exists in emitarm and emitxarch, so changing just for arm64 would be 'odd'.

Perhaps we should just leave this as you have it.

The imm isn't directly encoded, instead it is combined with the stack frame offset of the local variable.